ambry-cloud/src/main/java/com.github.ambry.cloud/CloudBlobStore.java

```diff
@@ -59,12 +61,20 @@
 class CloudBlobStore implements Store {
 
   private static final Logger logger = LoggerFactory.getLogger(CloudBlobStore.class);
+  private static final int cacheInitialCapacity = 1000;
+  private static final float cacheLoadFactor = 0.75f;
   private final PartitionId partitionId;
   private final CloudDestination cloudDestination;
   private final ClusterMap clusterMap;
   private final CloudBlobCryptoAgentFactory cryptoAgentFactory;
   private final CloudBlobCryptoAgent cryptoAgent;
   private final VcrMetrics vcrMetrics;
+
+  /** The lifecycle state of a recently seen blob. */
+  enum BlobState {CREATED, TTL_UPDATED, DELETED}
```

Comment:
Minior: move `BlobState` to `RecentBlobCache`?

Reply:
Can't because of dependency, but moved enum to bottom of class.